node/rpc-client/src/chain_ops.rs

```diff
@@ -0,0 +1,38 @@
+// Copyright 2020 ChainSafe Systems
+// SPDX-License-Identifier: Apache-2.0, MIT
+
+use super::client::Filecoin;
+use blocks::{header::json::BlockHeaderJson, tipset_json::TipsetJson};
+use cid::{json::CidJson, Cid};
+use jsonrpc_v2::Error as JsonRpcError;
+use jsonrpsee::raw::RawClient;
+use jsonrpsee::transport::http::HttpTransportClient as HTC;
+use message::unsigned_message::json::UnsignedMessageJson;
+
+/// Returns a block with specified CID fom chain via RPC
+pub async fn block(mut client: RawClient<HTC>, cid: Cid) -> Result<BlockHeaderJson, JsonRpcError> {
```

Comment:
Some CLI commands like FileCoin.SyncWait make multiple requests, so it might be better to have the client argument be a &mut in all of the *_ops.rs files